src/main/java/duke/DukeLauncher.java

```diff
@@ -0,0 +1,9 @@
+package duke;
+
+import javafx.application.Application;
+
+public class DukeLauncher {
+    public static void main(String[] args) {
+        Application.launch(Main.class, args);
+    }
```

Comment:
You might want to add javadocs for this class and for the public methods inside it

Reply:
It was a very recent addition. I'll add javadocs for the class and its public methods in the next push.